src/Core_Sideloader/UniversalAutoResolver/Core.UAR.cs

```diff
@@ -546,21 +554,21 @@ internal static void GenerateFaceSkinInfo(Manifest manifest, List<FaceSkinInfo>
         }
 #endif
 
-        internal static void ShowGUIDError(string guid)
+        internal static void ShowGUIDError(string guid, string author, string website)
         {
             Logging.LogLevel loglevel = Sideloader.MissingModWarning.Value ? Logging.LogLevel.Warning | Logging.LogLevel.Message : Logging.LogLevel.Warning;
 
             if (LoadedResolutionInfo.Any(x => x.GUID == guid))
                 //we have the GUID loaded, so the user has an outdated mod
-                Sideloader.Logger.Log(loglevel, $"[UAR] WARNING! Outdated mod detected! [{guid}]");
+                Sideloader.Logger.Log(loglevel, $"[UAR] WARNING! Outdated mod detected! [{guid}]:[By:{author}]:[Site:{website}]");
```

Comment:
The current message contains...unhelpful information, so time to read is irrelevant.

I guess if the mod author puts long stuff this could end up going off screen, not sure of a solution to that, could break them into separate messages? Having it in the text log for the user to find, or adding to third party tools like KKManager  is more useful.